ui/src/components/Document/DocumentTitle.js

```diff
@@ -3,12 +3,13 @@ import React from 'react';
 import Truncate from 'react-truncate';
 
 export const DocumentTitle = ({ document, short, truncate }) => {
+  const documentMetadata = document.metadata ? document.metadata : document;
   const title = short ? (
-    document.metadata.title
+    documentMetadata.title
   ) : (
     <>
-      {document.metadata.title} (<DocumentEdition document={document} />{' '}
-      {document.metadata.publication_year})
+      {documentMetadata.title} (<DocumentEdition document={document} />{' '}
```

Comment:
Is this `{' '}` needed? Just out of curiosity :)

Reply:
prettier formats it this way. it happens when there is a space and it wraps the text to new line

Reply:
Yes this I know, but is it needed this space?